pages/model/[uuid].tsx

```diff
@@ -27,11 +27,10 @@ import Stack from '@mui/material/Stack'
 import Tab from '@mui/material/Tab'
 import Tabs from '@mui/material/Tabs'
 import Typography from '@mui/material/Typography'
-import { useTheme } from '@mui/material'
+import { SelectChangeEvent, useTheme } from '@mui/material'
```

Comment:
We should try to rework this so we don't include `@mui/material`.  This alone increases our build times by 100% (well, at least when doing `import { Tab } from '@mui/material` instead of `import Tab from '@mui/material/Tab'`, I assume the same applies here).

Reply:
Good point, it should only affect dev build times, but I'll start changing any named mui imports to defaults.